src/System.Memory/src/System/MemoryExtensions.Fast.cs

```diff
@@ -3,6 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
```

Comment:
If we get rid of the AggressiveInlining, do we need `using System.Runtime.CompilerServices;` anymore now that we use `Internal.Runtime....` for Fast span?